src/en/mangarawclub/src/eu/kanade/tachiyomi/extension/en/mangarawclub/MangaRawClub.kt

```diff
@@ -159,8 +151,8 @@ class MangaRawClub : ParsedHttpSource(), ConfigurableSource {
 
     // Manga from Element
     override fun searchMangaFromElement(element: Element): SManga = SManga.create().apply {
-        title = element.selectFirst(".comic-card__title")!!.ownText()
-        thumbnail_url = element.select(".comic-card__cover img").attr("abs:data-src")
+        title = element.selectFirst(".comic-card__title a")!!.text().trim()
```

Comment:
```suggestion
        title = element.selectFirst(".comic-card__title a")!!.text()
```

`.text()` is already trimmed, according to Jsoup docs.